gum/backend-dbghelp/gumsymbolutil-dbghelp.c

```diff
@@ -133,7 +133,11 @@ gum_find_functions_matching (const gchar * str)
   if (dbghelp == NULL)
     return matches;
 
-  match_formatted_str = g_strdup_printf ("*!%s", str);
+
+  if (strstr (str, "!") == NULL)
```

Comment:
```suggestion
  if (strchr (str, '!') == NULL)
```

(Got lost in the noise of the earlier suggestion.)